packages/app/src/pages/MainPage.tsx

```diff
@@ -247,6 +269,44 @@ export const MainPage: React.FC<{}> = (props) => {
     downloadZKey();
   }, []);
 
+  const handleGenerateProofRemotely = async () => {
+    setIsRemoteProofGenerationLoading(true);
+    const input = await generateInputFromEmail(
+      "zk-email/proof-of-twitter-v2",
+      emailFull
+    );
+    const body = Buffer.from(input.emailBody).toString("utf-8");
+    console.log("input", input);
+    console.log(input);
+    try {
+      const proofRes = await generateProofRemotely(
+        "zk-email/proof-of-twitter-v2",
+        input
+      );
+    } catch (err) {
+      console.log("Something went wrong", err);
+      setIsRemoteProofGenerationLoading(false);
+    } finally {
+    }
+  };
```

Comment:
Will remove this part once I can add verify proof method in SDK. Currently, it is failing due to `AbiEncodingArrayLengthMismatchError` (messaged Javier about this in telegram)